content/chronograf/v1.9/guides/querying-data.md

```diff
@@ -1,6 +1,6 @@
 ---
 title: Explore data in Chronograf
-description: Query and visualize data in the Data Explorer.
+description: Query and visualize data in the Data Explorer
```

Comment:
We do want to keep the period here. This is used as the page description in search engine results.

Reply:
Done!  thanks for clarifying that